pkg/proxy/proxier.go

```diff
@@ -464,9 +617,13 @@ func iptablesCommonPortalArgs(destIP net.IP, destPort int, protocol api.Protocol
 		"--comment", service.String(),
 		"-p", strings.ToLower(string(protocol)),
 		"-m", strings.ToLower(string(protocol)),
-		"-d", fmt.Sprintf("%s/32", destIP.String()),
 		"--dport", fmt.Sprintf("%d", destPort),
 	}
+
+	if destIP != nil {
```

Comment:
I feel this code is pretty fragile, and it is nice to contain it in iptablesCommonPortalArgs.  So I'd lobby against doing this, but if you do really want it then I'll make the change.